package.json

```diff
@@ -68,7 +64,6 @@
     "lint-staged": "15.2.2",
     "npm-run-all": "4.1.5",
     "prettier": "3.2.5",
-    "ts-node": "^10.9.2",
```

Comment:
This breaks `Jest` run.

Reply:
Aha! I missed this one. Let me check this asap and fix Jest as well.